nixos/modules/services/hardware/fwupd.nix

```
@@ -86,6 +96,15 @@ in {
         '';
       };
 
+      extraRemotes = mkOption {
+        type = with types; listOf str;
+        default = [];
+        example = literalExpression ''[ "lvfs-testing" ]'';
```

I believe literals like this can be used literally:

```suggestion
        example = [ "lvfs-testing" ];
```